changelog/9514.bugfix.rst

```diff
@@ -0,0 +1 @@
+Type FixtureRequest.param as Any.
```

Comment:
Why is this a bug fix? Perhaps `improvement` or even `trivial` is better?

Also I think this reads a bit better:

```suggestion
Type-annotate ``FixtureRequest.param`` as ``Any`` as a stop gap measure until :issue:`8073` is fixed.
```